src/util/Images.ts

```diff
@@ -0,0 +1,37 @@
+const crypto = await import("crypto")
+const path = await import("path")
+const fs = await import("fs")
+const https = await import("https")
+
+const EXTERNAL_PATH = "./public/external/"
+
+// Download the image and return a local path if the image
+// is external
+export function image(src: string) {
+	if (!fs.existsSync(EXTERNAL_PATH)) {
+		fs.mkdirSync(EXTERNAL_PATH)
+	}
+
+	const url = new URL(src, "http://placeholder")
```

Comment:
I feel like doing this isn't the *best* way of doing it... is there no alternative (e.g. checking the scheme of the URL)?